content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md

```diff
@@ -2,3 +2,5 @@
 title: Create a Cluster
 weight: 10
 ---
+
+{{< glossary_definition term_id="cluster" length="short" >}}
```

Comment:
In spite of adding this line, the description for "cluster" does not appear.

---

[As-is]
https://kubernetes.io/docs/tutorials/kubernetes-basics/create-cluster/
![image](https://user-images.githubusercontent.com/46767780/125229107-02736c00-e311-11eb-90fa-e8224b578f32.png)

---

[Preview of this PR]
https://deploy-preview-28905--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/create-cluster/
![image](https://user-images.githubusercontent.com/46767780/125229145-13bc7880-e311-11eb-9f5e-3d7651dfdd2c.png)

Reply:
Thanks for the reviews! I will update accordingly.

Reply:
Updated the PR. PTAL.